src/App.js

```diff
@@ -1,25 +1,62 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import axios from 'axios';
+import './App.css'
+import { Jumbotron, Image, Form, Button } from 'react-bootstrap';
+import ErrorModal from './components/errorModal';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      imgSrc: '',
+      displayResults: false,
+      displayErrorModal: false,
+      errorMessage: '',
+    }
+  }
+
+  getLocationInfo = async (e) => {
+    e.preventDefault();
+    const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATION_KEY}&q=${this.state.searchQuery}&format=json`;
+    await axios.get(url)
+      .then(location =>
+        this.setState({
+          location: location.data[0],
+          displayResults: true,
+          imgSrc: `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATION_KEY}&center=${location.data[0].lat},${location.data[0].lon}&zoom=13`
+        }))
+      .catch(err => {
+        console.log(err.message);
+        this.setState({ displayErrorModal: true, errorMessage: err.message })
+      });
+  }
+
+  hideErrorModal = () => {this.setState({ displayErrorModal: false })}
+
+  render() {
+    return (
+      <div className="App">
+        <h1>City Explorer</h1>
+        <Form onSubmit={this.getLocationInfo}>
+          <Form.Group>
+            <Form.Control onChange={(e) => this.setState({ searchQuery: e.target.value })} placeholder="City"/>
+            <Form.Text className="text-muted">Enter a city name to learn a bit more about it.</Form.Text>
+          </Form.Group>
+          <Button variant="primary" type="submit">Explore!</Button>
+        </Form>
+        {this.state.displayResults &&
```

Comment:
Make Jumptron into a component -> you could also split up those into their own components